Index: work/src/C/sparse.c
===================================================================
--- work.orig/src/C/sparse.c
+++ work/src/C/sparse.c
@@ -426,7 +426,7 @@ if (!A) return NULL;
                 SP_COL(A)[nk+1]++;
               }
             }
-          } else if SpMatrix_Check(Lij) {
+          } else if (SpMatrix_Check(Lij)) {

             int_t ik;
             for (ik=SP_COL(Lij)[jk]; ik<SP_COL(Lij)[jk+1]; ik++) {
Index: work/src/C/base.c
===================================================================
--- work.orig/src/C/base.c
+++ work/src/C/base.c
@@ -353,7 +353,7 @@ int (*mtx_rem[])(void *, number, int) =
 /* val_type = 0: (sp)matrix if type = 0, PY_NUMBER if type = 1 */
 int get_id(void *val, int val_type) {
   if (!val_type) {
-    if Matrix_Check((PyObject *)val)
+    if (Matrix_Check((PyObject *)val))
     return MAT_ID((matrix *)val);
     else
       return SP_ID((spmatrix *)val);
